scenarioo-client/app/shared/navigation/sharePage/sharePageUrl.service.ts

```diff
@@ -0,0 +1,44 @@
+import {Injectable} from '@angular/core';
+import {downgradeInjectable} from '@angular/upgrade/static';
+
+declare var angular: angular.IAngularStatic;
+
+@Injectable()
+export class SharePageURL {
+
+    constructor() {
+        this.shareLinks = {
```

Comment:
use ngOnInit

Reply:
ngOnInit does only work in Components and Directives, not in Services.